src/neophile/pr.py

```diff
@@ -199,6 +199,32 @@ async def _get_github_actor(self) -> Actor:
         else:
             return Actor(response["name"], response["email"])
 
+    async def _get_github_main_branch(
+        self, github_repo: GitHubRepository
+    ) -> str:
+        """Get the main branch of the repository.
+
+        Uses ``main`` if that branch exists, else ``master``.
+
+        Parameters
+        ----------
+        github_repo : `neophile.config.GitHubRepository`
+            GitHub repository in which to create the pull request.
+
+        Returns
+        -------
+        branch : `str`
+            The name of the main branch.
+        """
+        branches = self._github.getiter(
+            "/repos{/owner}{/repo}/branches",
+            url_vars={"owner": github_repo.owner, "repo": github_repo.repo},
+        )
+        async for branch in branches:
```

Comment:
It seems like it would be cleaner to call to `/repos{/owner}{/repo}` and then read the value of `default_branch` out of the response.  That way we don't have to hard code the acceptable default branches.

Reply:
Oh, good call, I missed that API. New diff coming.